test/unit/io/sequence_file/sequence_file_format_embl_test.cpp

```diff
@@ -119,6 +119,9 @@ SQ Sequence 1859 BP; 609 A; 314 C; 355 G; 581 T; 0 other;
         {
             EXPECT_EQ(seqan3::get<seqan3::field::id>(*it), ids[i]);
             EXPECT_EQ(seqan3::get<seqan3::field::seq>(*it), seqs[i]);
+
+            EXPECT_EQ((*it).id(), ids[i]);
```

Comment:
We discussed that just yesterday :)

It is not guaranteed that C++20 iterators have a pointer access operator (they don't require that). The reason is simple, you could not give back a temporary object, like a proxy, if you would require that. So the safest way, since C++20 is to write `(*it).member` instead of `it->member`.